examples/OpNavScenarios/plottingOpNav/OpNav_Plotting.py

```diff
@@ -743,7 +743,7 @@ def imgProcVsExp(true, centers, radii, size):
         pass
     plt.ylabel('X (px)')
     plt.xlabel('Time (min)')
-    plt.grid(b=None, which='major', axis='y')
```

Comment:
We're changing functionality here. The old ``b`` command would turn off the gridlines. If we remove it, gridlines will likely appear. According to the online documentation, we should change it to ``visible=None``.

Reply:
As there is an API change here, and these lines are really not needed, I'm just going to delete these particular grid line commands.  They would just cause issues between current and prior versions of matplotlib.  No functional reason to have these commands in these plots.